File: ExploringPlanetsUI/src/app/ModelsDTO/registerDTO.ts
export class Register{
    public Id: string | undefined;
    public FirstName: string | undefined;
    public LastName: string | undefined;
    public Email: string | undefined;
    public Password: string | undefined;
    public Role: string | undefined;

    constructor(init?: Partial<Register>){
        Object.assign(this, init);
        this.Role = 'Captain';
    }
}

File: ExploringPlanets/ObjectsDTO/RegisterDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ExploringPlanets.ObjectsDTO
{
    public class RegisterDTO
    {
        public string Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string Role { get; set; }
    }
}
